src/sentry/web/urls.py

```diff
@@ -598,12 +597,6 @@
         VstsExtensionConfigurationView.as_view(),
         name="vsts-extension-configuration",
     ),
-    # vercel Marketplace extension install flow
-    url(
-        r"^extensions/vercel/configure/$",
-        VercelExtensionConfigurationView.as_view(),
```

Comment:
Realized it was cleaner to just add this route to the Vercel specific router. Will do the same change with VSTS later.